<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LocalTester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Serial Port:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="serialPort"/>
    </item>
    <item>
     <widget class="QPushButton" name="refreshSerialPorts">
      <property name="text">
       <string>Refresh Serial Ports</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Relay IP Address(unused for local testing):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="relayIp"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Send/Test Args:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="sendText"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Receive:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="receiveText">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Screen:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="framebuffer">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::WinPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pickDependencyBlob">
      <property name="text">
       <string>Pick Dependency Blob</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pickTestProgram">
      <property name="text">
       <string>Pick Test Program</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pickSslCertificate">
      <property name="text">
       <string>Pick SSL Certificate(only needed for remote debugging)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startLocalTesting">
      <property name="text">
       <string>Start Local Testing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startRemoteTesting">
      <property name="text">
       <string>Start Remote Testing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="startTestProxy">
      <property name="text">
       <string>Start Test Proxy</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
